src/runtime/hexagon/README.md

```diff
@@ -0,0 +1,76 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Hexagon backend runtime
+
+The Hexagon runtime is a part of the TVM runtime that facilitates communication between a host and a Hexagon device. There are two types of host/device arrangements that are supported:
+- X86/Linux host running Hexagon simulator,
+- Android/AArch64 host running on a physical device containing a Hexagon module (i.e. CSDP or ADSP).
+
+The TVM runtime that contains Hexagon runtime is the one executing on host.  In either case, there will need to be a separate TVM runtime (i.e.  the `libtvm_runtime.so` library) compiled for execution on Hexagon.
+
+The prerequisite is to have Hexagon SDK installed, preferably version 3.5.0 or later. The Hexagon SDK can be downloaded from https://developer.qualcomm.com/software/hexagon-dsp-sdk.
+
+### Compiling TVM runtime for x86
+
+This will use Hexagon simulator, which is provided in the Hexagon SDK.
+
+When configuring TVM (cmake), set the following variables:
+```
+USE_LLVM=llvm-config
+USE_HEXAGON_DEVICE=sim
+USE_HEXAGON_SDK=/path/to/sdk
+```
+
+Set the C/C++ compiler to `clang`, and pass `-DCMAKE_CXX_FLAGS='-stdlib=libc++'` to the cmake command.
```

Comment:
When we started, I was seeing some problems with libstdc++, but I don't remember what they were anymore.  Since it worked for you, I removed this line.